src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -2341,32 +2365,38 @@ namespace SystemAvailabilityManager {
 				OptStartData.OccStartTime.allocate( NumOfZones );
 			}
 			if ( ! allocated( OptStartData.ActualZoneNum ) ) OptStartData.ActualZoneNum.allocate( NumOfZones );
-			OptStartData.OptStartFlag = false;
-			OptStartData.OccStartTime = 99.99; //initialize the zone occupancy start time
+			if ( BeginDayFlag ) {
+				NumHoursBeforeOccupancy = 0.0; //Initialize the hours of optimum start period. This variable is for reporting purpose.
+				OptStartData.OccStartTime = 22.99; //initialize the zone occupancy start time
+				OptStartData.OptStartFlag = false;
+			}
+
```

Comment:
The OptStartData needs to be reset each day. Can get rid of reset for NumHoursBeforeOccupancy, I think.